dockers/skylab/atac_barcodes/atac_barcodes.py

```diff
@@ -0,0 +1,43 @@
+#! /usr/bin/env python
+
+# Script to append R2 barcodes to ATAC read 1 and read 3 FASTQs
+from argparse import ArgumentParser
+from itertools import islice
+
+def write_fastq(args):
```

Comment:
```suggestion
def write_fastq(args):
"""
 
"""
```

Reply:
I think it might read better to indent the docstring to be on the same level of the commands in the function